packages/ckeditor5-paste-from-office/tests/_data/list/resume-template/normalized.safari.word2016.html

```diff
@@ -1,198 +1,170 @@
-<w:sdt docpart="FDD2594584A640499F749D0B4EC7B4EF" id="1524743180" sdttag="Your Name:" showingplchdr="t" style="-webkit-text-size-adjust:auto;-webkit-text-stroke-width:0px;caret-color:rgb(0, 0, 0);color:rgb(0, 0, 0);font-family:-webkit-standard;font-style:normal;font-variant-caps:normal;font-weight:normal;letter-spacing:normal;orphans:auto;text-align:start;text-decoration:none;text-indent:0px;text-transform:none;white-space:normal;widows:auto;word-spacing:0px" temporary="t" title="Your Name:">
```

Comment:
Maybe we should somehow transfer some legitimate styles to the children while unwrapping?

Reply:
Style of this `SdtBlock` looks like some kind of set of default values of standard `CSS` attributes. As far as i can see we can't move them to the children cause they can contains same style attributes but with different values, so this could break whole style of the child (children).

Maybe other solution is to change `<w:sdt>` tag into `<div>` remove all attributes except `style`?? But don't know what side effects would it bring. WDYT?

Reply:
> Maybe other solution is to change `<w:sdt>` tag into `<div>` remove all attributes except `style`?? But don't know what side effects would it bring. WDYT?

This could break the layout if `sdt` element is inside a paragraph.

Reply:
> This could break the layout if `sdt` element is inside a paragraph.

Yes, it can be inside paragraph, span or other.

Maybe would be good to ask someones from `CERD` tram? Maybe they will know if we can drop this without any consequences :thinking: